dat.py

```diff
@@ -188,32 +197,41 @@ def get(self, gene_str):
 
 
 if __name__ == '__main__':
-    from optparse import OptionParser
-
-    usage = "usage: %prog [options]"
-    parser = OptionParser(usage, version="%prog dev-unreleased")
-    parser.add_option("-i", "--dab-file", dest="dab", help="DAB file",
-                      metavar="FILE")
-    parser.add_option("-o", "--output-file", dest="out",
-                      help="Output file (DAT or PCL)", metavar="FILE")
-
-    (options, args) = parser.parse_args()
+    from argparse import ArgumentParser
+
+    usage = "usage: %(prog)s [options]"
+    parser = ArgumentParser(prog=usage)
+    parser.add_argument("-i", "--dab-file", dest="dab", help="DAB file",
+                        metavar="FILE")
+    parser.add_argument("-o", "--output-file", dest="out",
+                        help="Output file (DAT or PCL)", metavar="FILE")
+    parser.add_argument("-v", "--verbose", dest="verbose", action='store_true',
+                        help="output debug loglevel")
+    parser.add_argument('-V', '--version', action='version',
+                        version="%(prog)s dev-unreleased")
+    args = parser.parse_args()
+    print(args)
```

Comment:
If you paste the whole commit hash (`a094f575e170e8669683277625911749e560312d`) it will be auto-hyper-linked like a094f575e170e8669683277625911749e560312d.